extensions.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We need .csproj files in a Node.js function? :-)

Reply:
Yes we do now. Since trigger binding in `~2` are now installed as a separate extensions to your function. In `~1` is was managed automatically.